toqito/matrix_props/positive_semidefinite_rank.py

```diff
@@ -0,0 +1,87 @@
+"""Positive semidefinite rank of a nonnegative matrix."""
+
+import cvxpy as cp
+import numpy as np
+from toqito.matrix_props import is_square
+
+
+def positive_semidefinite_rank(mat: np.ndarray, max_rank: int = 10) -> int | None:
+    r"""Compute the positive semidefinite rank (PSD rank) of a nonnegative matrix.
+
+    The definition of PSD rank is defined in :cite:`Fawzi_2015_Positive`.
```

Comment:
![image](https://github.com/vprusso/toqito/assets/66048318/fd04a9fc-cd21-4af6-8ce8-c54b8888d2ea)

![image](https://github.com/vprusso/toqito/assets/66048318/fb384925-eb5b-4851-bde4-f2a128594748)


These show up as a blank entries in the RTD build. I think this is due to the `bib` entry not having a URL in the description. 

https://toqito--647.org.readthedocs.build/en/647/autoapi/matrix_props/positive_semidefinite_rank/index.html

Reply:
Fixed!